### PR TITLE
feat: clean up storybook

### PR DESCRIPTION
**Pull request by mikasackermn**
# Summary

Clean up Storybook.

# Checklist:

- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by samobasquiat** [comment]
Can we remove these files as well?
`widget/ui/src/components/Tooltip/useEventListener.ts`
`widget/ui/src/components/Wallet/Content.tsx`
`widget/ui/src/icons/Gass.tsx`
`widget/ui/src/types/meta.ts`

Also `rgb-hex` from package.json (I don't know about react.is. is it necessary? )
In addition, I think we should add `copy-to-clipboard` package to package json.

moreover:
```
FirstImage                            unknown  src/components/SelectableCategoryList/SelectableCategoryList.styles.ts:16:14
ImageContent                          unknown  src/components/SelectableCategoryList/SelectableCategoryList.styles.ts:9:14 
LoadingContainer                      unknown  src/components/SwapListItem/SwapToken.styles.ts:71:14 
```

**Comment by samobasquiat** [comment]
In most cases, removing marginLeft and adding Divider caused  Ui bugs. The divider is added to the right, not the left.image

File: widget/ui/src/components/StepDetails/StepDetails.tsx [hunk 1]
```diff
@@ -95,7 +96,7 @@ const StepDetailsComponent = forwardRef<HTMLDivElement, StepDetailsProps>(
                     <SwapperImage state={state}>
                       <Image size={22} src={swapperItem.swapper.image} />
                     </SwapperImage>
-                    <Typography ml={8} size="medium" variant="label">
+                    <Typography size="medium" variant="label">
```
Comment (samobasquiat) [inline widget/ui/src/components/StepDetails/StepDetails.tsx:99]: .

File: widget/ui/src/components/TokenAmount/TokenAmount.tsx [hunk 2]
```diff
@@ -24,29 +25,33 @@ export function TokenAmount(props: PropTypes) {
...
         <div>
           {props.label && (
-            <Typography ml={4} size="xsmall" variant="body" color="$neutral700">
-              {props.label}
-            </Typography>
+            <>
+              <Typography size="xsmall" variant="body" color="$neutral700">
+                {props.label}
+              </Typography>
+              <Divider direction="horizontal" size={4} />
```
Comment (samobasquiat) [inline widget/ui/src/components/TokenAmount/TokenAmount.tsx:32]: .

File: widget/ui/src/components/TokenAmount/TokenAmount.tsx [hunk 3]
```diff
@@ -24,29 +25,33 @@ export function TokenAmount(props: PropTypes) {
...
               open={!props.price.realValue ? false : undefined}
               container={props.tooltipContainer}>
               <Typography
-                ml={4}
                 size="medium"
                 variant="title"
                 style={{ fontWeight: 600 }}>
                 {props.price.value}
               </Typography>
+              <Divider direction="horizontal" size={4} />
```
Comment (samobasquiat) [inline widget/ui/src/components/TokenAmount/TokenAmount.tsx:46]: .

File: widget/ui/src/components/TokenAmount/TokenAmount.tsx [hunk 4]
```diff
@@ -24,29 +25,33 @@ export function TokenAmount(props: PropTypes) {
...
+
               <Typography
-                ml={8}
                 size="medium"
                 variant="title"
                 style={{ fontWeight: 400 }}>
                 {props.token.displayName}
               </Typography>
+              <Divider direction="horizontal" size={8} />
```
Comment (samobasquiat) [inline widget/ui/src/components/TokenAmount/TokenAmount.tsx:54]: .

File: widget/ui/src/components/TokenAmount/TokenAmount.tsx [hunk 5]
```diff
@@ -57,11 +62,7 @@ export function TokenAmount(props: PropTypes) {
             content={props.price.realUsdValue}
             container={props.tooltipContainer}>
             {props.type === 'input' && (
-              <Typography
-                mr={4}
-                size="small"
-                variant="body"
-                color="$neutral700">
+              <Typography size="small" variant="body" color="$neutral700">
```
Comment (samobasquiat) [inline widget/ui/src/components/TokenAmount/TokenAmount.tsx:65]: .

File: widget/ui/src/containers/SwapInput/SwapInput.tsx [hunk 6]
```diff
@@ -47,11 +47,7 @@ export function SwapInput(props: SwapInputProps) {
           </Typography>
           {showBalance && (
             <div className={balanceStyles()}>
-              <Typography
-                className={textStyles()}
-                mr={4}
-                variant="body"
-                size="xsmall">
+              <Typography className={textStyles()} variant="body" size="xsmall">
```
Comment (samobasquiat) [inline widget/ui/src/containers/SwapInput/SwapInput.tsx:50]: .

**Review by samobasquiat — CHANGES_REQUESTED** [review]

File: widget/embedded/src/pages/SelectBlockchainPage.tsx [hunk 7]
```diff
@@ -33,6 +33,7 @@ export function SelectBlockchainPage(props: PropTypes) {
   const countActiveCategories = getCountCategories(blockchains);
 
   const showCategory = countActiveCategories !== 1;
+  console.log(blockchains);
```
Comment (samobasquiat) [inline widget/embedded/src/pages/SelectBlockchainPage.tsx:36]: Please remove it.

File: widget/ui/src/components/TokenAmount/TokenAmount.tsx (hunk 2 above)
Comment (samobasquiat) [reply widget/ui/src/components/TokenAmount/TokenAmount.tsx:32]: I think It should be added to left

File: widget/embedded/src/pages/SelectBlockchainPage.tsx (hunk 7 above)
Comment (samobasquiat) [reply widget/embedded/src/pages/SelectBlockchainPage.tsx:36]: !

File: widget/ui/src/components/StepDetails/StepDetails.tsx [hunk 8]
```diff
@@ -151,13 +154,12 @@ const StepDetailsComponent = forwardRef<HTMLDivElement, StepDetailsProps>(
...
-                  color="$neutral700"
-                  variant="body">{`${isCompleted ? '' : '~'}${
-                  step.to.price.value
-                } ${step.to.token.displayName}`}</Typography>
+                <Typography size="small" color="$neutral700" variant="body">{`${
+                  isCompleted ? '' : '~'
+                }${step.to.price.value} ${
+                  step.to.token.displayName
+                }`}</Typography>
+                <Divider direction="horizontal" size={4} />
```
[the remark this answers is not in the document]
Comment (samobasquiat) [reply widget/ui/src/components/StepDetails/StepDetails.tsx:162]: I think It should be added to left

File: widget/ui/src/components/TokenAmount/TokenAmount.tsx (hunk 3 above)
Comment (samobasquiat) [reply widget/ui/src/components/TokenAmount/TokenAmount.tsx:46]: image

File: widget/ui/src/components/StepDetails/StepDetails.tsx (hunk 8 above)
[the remark this answers is not in the document]
Comment (samobasquiat) [reply widget/ui/src/components/StepDetails/StepDetails.tsx:162]: image

**Review by samobasquiat — APPROVED** [review]

File: widget/ui/src/components/TokenAmount/TokenAmount.tsx [hunk 9]
```diff
@@ -24,24 +25,30 @@ export function TokenAmount(props: PropTypes) {
         />
         <div>
           {props.label && (
-            <Typography ml={4} size="xsmall" variant="body" color="$neutral700">
-              {props.label}
-            </Typography>
+            <div style={{ display: 'flex' }}>
```
Comment (yeager-eren) [inline widget/ui/src/components/TokenAmount/TokenAmount.tsx:28]: This looks a working around to me. I think `Divider` should works here correctly (if not, we should fix that component) or it should be fixed somehow in layout layer of this component.

Comment (yeager-eren) [reply widget/ui/src/components/TokenAmount/TokenAmount.tsx:28]: There are some other places that has this working around, Same comment applies to them as well. I don't put comments on them to keep comments cleaner.  Please make sure you're checking them as well.

File: widget/ui/src/components/RefreshProgressButton/RefreshProgressButton.stories.tsx [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+import React from 'react';
+
+import { RefreshProgressButton } from './RefreshProgressButton';
+
+export default {
+  title: 'Components/RefreshProgressButton',
+  component: RefreshProgressButton,
+  args: {
+    size: 22,
+    color: 'black',
```
Comment (yeager-eren) [inline widget/ui/src/components/RefreshProgressButton/RefreshProgressButton.stories.tsx:13]: I'm not sure this is related to you changes or not, but changing `color` value to other colors, doesn't work.

**Review by yeager-eren — CHANGES_REQUESTED** [review]
Thanks. Feels good to get rid of unused codes.

File: widget/ui/src/components/Tabs/Tabs.styles.tsx [hunk 11]
```diff
@@ -155,3 +155,8 @@ export const BackdropTab = styled('div', {
     },
   },
 });
+
+export const TabsContainer = styled('div', {
```
[the remark this answers is not in the document]
Comment (mikasackermn) [reply widget/ui/src/components/Tabs/Tabs.styles.tsx:159]: You're right, of course. I move this to the file for the storybook.

File: widget/ui/src/components/RefreshProgressButton/RefreshProgressButton.stories.tsx (hunk 10 above)
Comment (mikasackermn) [reply widget/ui/src/components/RefreshProgressButton/RefreshProgressButton.stories.tsx:13]: I checked it over and there was nothing wrong.
Keep in mind that the props in the documents section cannot be changed.

File: widget/ui/src/components/TokenAmount/TokenAmount.tsx (hunk 9 above)
Comment (mikasackermn) [reply widget/ui/src/components/TokenAmount/TokenAmount.tsx:28]: I wasn't really sure what this meant. Are you saying that the divider itself should to support the display flex?

**Review by yeager-eren — APPROVED** [review]
Thanks. lgtm.

**Pull request merged**